generated/nidaqmx/_grpc_interpreter.py

```diff
@@ -3547,4 +3543,15 @@ def _assign_numpy_array(numpy_array, grpc_array):
 def _validate_array_dtype(numpy_array, expected_numpy_array_dtype):
     """Raises TypeError if array type doesn't match with expected numpy.dtype"""
     if expected_numpy_array_dtype != numpy.generic and numpy_array.dtype != expected_numpy_array_dtype:
-        raise TypeError(f"array must have data type {expected_numpy_array_dtype}")
\ No newline at end of file
+        raise TypeError(f"array must have data type {expected_numpy_array_dtype}")
+
+def _is_cancelled(ex: Exception):
+    """Returns True if the given exception is a cancelled RPC exception."""
+    if(isinstance(ex, grpc.RpcError)):
+        if ex.code() == grpc.StatusCode.CANCELLED:
+            return True
+    elif (isinstance(ex, errors.RpcError)):
+        if ex.rpc_code == grpc.StatusCode.CANCELLED:
+            return True
+    
+    return False
```

Comment:
[style] if-statements don't need parens, replace "if x: if y:" with "if x and y:", replace "if x: return True" with "return x"

```suggestion
    return (
        (isinstance(ex, grpc.RpcError) and ex.code() == grpc.StatusCode.CANCELLED)
        or (isinstance(ex, errors.RpcError) and ex.rpc_code == grpc.StatusCode.CANCELLED)
    )
```

Reply:
Updated as per suggestions